doc/lua_api.txt

```diff
@@ -8474,44 +8493,198 @@ Used by `minetest.register_node`.
 Crafting recipes
 ----------------
 
-Used by `minetest.register_craft`.
+All crafting recipes take one or more items as input and have an
+optional output.
+Crafting needs inventory slots. From the crafting inputs, only one
```

Comment:
> Crafting needs inventory slots. 

What do you mean by that? `minetest.get_craft_result` for example does not need inventory slots, and the commonly known act of "crafting" obviously needs space to work with.